modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -1,31 +1,17 @@
 # frozen_string_literal: true
 
 require 'sidekiq'
-require 'va_forms/regex_helper'
 
 module VAForms
   class FormReloader
     include Sidekiq::Worker
-    include SentryLogging
-    FORM_BASE_URL = 'https://www.va.gov'
 
     def perform
       Rails.logger.info("#{self.class.name} is being called.")
-      query = File.read(Rails.root.join('modules', 'va_forms', 'config', 'graphql_query.txt'))
-      body = { query: }
-      response = connection.post do |req|
-        req.path = 'graphql'
-        req.body = body.to_json
-        req.options.timeout = 300
-      end
-      forms_data = JSON.parse(response.body)
-      forms_data.dig('data', 'nodeQuery', 'entities').each do |form|
-        build_and_save_form(form)
-      rescue => e
-        message = "#{self.class.name} failed to import #{form['fieldVaFormNumber']} into forms database"
-        Rails.logger.error(message, e)
-        log_message_to_sentry(message, :error, body: e.message)
-        next
+
+      builder = VAForms::FormBuilder.new
+      all_forms_data.each do |form|
+        builder.perform_async(form)
```

Comment:
Not a blocker, but with so many jobs being spawned (something like 700 IIRC), we may need to implement a [delay strategy](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/workers/appeals_api/notice_of_disagreement_upload_status_batch.rb#L18-L20) to ensure we don't overload servers w/ too many requests.

Also, I'm not sure you need an instance of the builder job - most the calls to perform_async I've seen have been as a class method and not an instance method